src/main/studio/kdb/Config.java

```diff
@@ -670,7 +670,9 @@ public void setDefaultCredentials(String authenticationMechanism, Credentials cr
     }
 
     public String getDefaultAuthMechanism() {
-        return p.getProperty("auth", DefaultAuthenticationMechanism.NAME);
+        String result = p.getProperty("auth", null);
+        if (result == null) result = System.getenv().getOrDefault("KDB_STUDIO_AUTH_METHOD", DefaultAuthenticationMechanism.NAME);
+        return result;
```

Comment:
The env. variable will be used until the user opens Setting dialog. If the user change the env. variable after that, it wouldn't take any effect. Is it desired behaviour?

Reply:
The original use case is to just override the default for the first time the user starts the app, then it should be saved in the settings. This saves the user the step of changing the default auth method manually (or worse, not realizing that there is a way to set the default and so setting it on each server), and eliminates the support requests from users failing to do so.